src/components/PayPalMarks.test.tsx

```diff
@@ -137,17 +137,19 @@ describe("<PayPalMarks />", () => {
         const spyConsoleError = jest
             .spyOn(console, "error")
             .mockImplementation();
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        window.paypal!.Marks = () => {
-            return {
-                isEligible: jest.fn().mockReturnValue(true),
-                render: jest.fn((element) => {
-                    // simulate adding markup for paypal mark
-                    if (typeof element != "string")
-                        element.append(document.createElement("div"));
-                    return Promise.reject("Unknown error");
-                }),
-            };
+        window.paypal = {
+            Marks() {
+                return {
+                    isEligible: jest.fn().mockReturnValue(true),
+                    render: jest.fn((element) => {
+                        // simulate adding markup for paypal mark
+                        if (typeof element != "string")
```

Comment:
Please wrap `if` in brackets
```suggestion
                        if (typeof element != "string") {
                            element.append(document.createElement("div"));
                        }
```